notebooks/Gemma+Gemini+Eval.ipynb

```diff
@@ -0,0 +1,8838 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    def find_json_snippet(raw_snippet):
</pre><p>What's the purpose of this thing? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/llmops-pipeline/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Parsing JSON bits from the output. Sometimes, it emits with and without ``` or ```json. To make sure if there is passable JSON, I am manually finding that out with this simple method. Have been using it for a while, and it worked on all cases well